src/main/java/racing/Racing.java

```diff
@@ -0,0 +1,38 @@
+package racing;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Racing {
+
+    private final List<Car> cars;
+    private final Round round;
+    private final List<RoundResult> results;
+
+    public Racing(RacingInfo info){
+        cars = info.getCars();
+        round = info.getRound();
+        results = new ArrayList<>();
+    }
+
+    public void race() {
+        int roundNumber = round.getRoundNumber();
+
+        while (roundNumber-- > 0) {
```

Comment:
```suggestion
        while (runGame() > 0) {
```
가독성을 위해 기능을 분리해보는 것은 어떨까요?